cmd/kubeadm/app/util/initsystem/initsystem.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 package initsystem
 
+// InitSystem is the interface that describe behaviors of an init system
```

Comment:
So we are the only place that used this library?

Reply:
Yes